mentoring-wg/gsoc-org-admin-guide.md

```diff
@@ -0,0 +1,167 @@
+* Program evaluation: Admins evaluate the success of the program and identify areas for improvement. They do this by analyzing program data, collecting feedback from contributors and mentors, and making recommendations for future iterations of the program.
+
+Following are not responsibilities of a GSoC admin:
+* Verifying the eligibilities of contributors in terms of age, country of residence and similar criteria. These are done by Google.
+* Providing technical support to mentors or contributors about the project.
+* Technically mentoring the contributors (e.g. technical troubleshooting), other than giving general guidance and/or replying to questions by pointing at possible places to find an answer.
+* Providing non-technical mentoring to the contributors (e.g. career advice, personal advice, etc.).
+
+## Checklist
+
+### GSoC program announcement
+
+At this stage, Google announces that they will have GSoC in the upcoming year. There is no guarantee from Google that CNCF will be accepted to GSoC.
+
+Tasks:
+* Reach out to CNCF board/TOC to get approval for applying to GSoC.
+* Create GSoC ideas page in [`cncf/mentoring`](https://github.com/cncf/mentoring/tree/main/summerofcode) repository. Ideas page will be necessary during organization application.
+* [Announce](#Announcements) the intention to participate in the program. Mention these:
+    * Deadline to add ideas is the contributor application period start date.
+    * CNCF has not yet been accepted into the program.
+    * We are collecting ideas.
+    * We are encouraging communities to add ideas earlier rather than later to give potential contributors more time.
+* Start monitoring PRs to ideas page, review ideas added by mentors using the [proposal review guideline](#Project-idea-proposal-review-guidelines).
+
+### Pre- organization application period
+
+At this stage, there is still no guarantee from Google that CNCF will be accepted to GSoC. CNCF is collecting ideas for potential contributors and preparing for the organization application to GSoC.
+
+Tasks:
+
+* Send a reminder to CNCF project communities to add ideas as stated in the previous stage.
+
+
+### Organization application
+
+At this stage, there should be an ideas page with a few ideas listed.
+
+Tasks:
+* Apply to the program.
+
+### Acceptance announcement
+
+* [Announce](#Announcements) the acceptance in the program. Mention these:
+    * CNCF is accepted to the program.
+    * It is still possible to add ideas until the contributor application period start date.
+    * We are encouraging communities to add ideas earlier rather than later to give potential contributors more time.
+    * We are encouraging contributors to explore project ideas and engage with mentors and communities.
```

Comment:
```suggestion
    * We encourage communities to add ideas earlier rather than later to give potential contributors more time.
    * We encourage contributors to explore project ideas and engage with mentors and communities.
```